context/context.go

```diff
@@ -78,8 +77,6 @@ func InitNssfContext() {
 	nssfContext.RegisterIPv4 = nssfConfig.Configuration.Sbi.RegisterIPv4
 	nssfContext.SBIPort = nssfConfig.Configuration.Sbi.Port
 	nssfContext.BindingIPv4 = os.Getenv(nssfConfig.Configuration.Sbi.BindingIPv4)
-	nssfContext.Key = util.NSSF_KEY_PATH // default key path
```

Comment:
is this not used anywhere?

Reply:
No, it is not. It is just a "placeholder" when using TLS. Actually, in the following lines (80-87) is where the actual path to the files are passed/set
```go
	if tls := nssfConfig.Configuration.Sbi.TLS; tls != nil {
		if tls.Key != "" {
			nssfContext.Key = tls.Key
		}
		if tls.PEM != "" {
			nssfContext.PEM = tls.PEM
		}
	}
```